### PR TITLE
New package: kitty 2.6.1

### PR DESCRIPTION
**Pull request by ibrokemypie**

### CODE REVIEW

**Comment by Duncaen** [comment]
You cannot simply update `glew`, you need to bump the revision and rebuild every package that depends on it.

File: srcpkgs/kitty/template [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Template file for 'kitty'
+pkgname=kitty
+version=0.2.6
+revision=1
+#only_for_archs="i686 x86_64"
+nopie=true
+python_version=3
+hostmakedepends="gcc pkg-config"
```
Comment (Vaelatern) [inline srcpkgs/kitty/template]: no `gcc`

File: srcpkgs/kitty/template [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Template file for 'kitty'
+pkgname=kitty
+version=0.2.6
+revision=1
+#only_for_archs="i686 x86_64"
+nopie=true
```
Comment (Vaelatern) [inline srcpkgs/kitty/template]: why? Probably not wanted or good.

File: srcpkgs/kitty/template [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# Template file for 'kitty'
+pkgname=kitty
+version=0.2.6
+revision=1
+#only_for_archs="i686 x86_64"
```
Comment (Vaelatern) [inline srcpkgs/kitty/template]: remove comment

File: srcpkgs/kitty/template [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+nopie=true
+python_version=3
+hostmakedepends="gcc pkg-config"
+makedepends="python3 python3-devel ncurses glew-devel fontconfig-devel glfw-devel"
```
Comment (Vaelatern) [inline srcpkgs/kitty/template]: `python3-devel` depends on `python3` so remove the redundancy.

File: srcpkgs/kitty/template [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+maintainer="ibrokemypie <ibrokemypie@bastardi.net>"
+license="GPL-3"
+homepage="https://github.com/kovidgoyal/kitty"
+distfiles="https://github.com/kovidgoyal/kitty/archive/v0.2.6.tar.gz"
+checksum=fe0922ce1b9f46e7b5402b9348b462d5f91a185b442e3f703940d01dea2fab21
+
+do_build(){
+	#python3 setup.py build
+	python3 setup.py --prefix=#{DESTDIR}/usr build
+}
```
Comment (Vaelatern) [inline srcpkgs/kitty/template]: python3 build style should be trusted with `do_build`

File: srcpkgs/kitty/template [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+
+do_install(){
+	python3 setup.py --prefix=${DESTDIR}/usr linux-package
+}
```
Comment (Vaelatern) [inline srcpkgs/kitty/template]: Also `do_install`

**Comment by ibrokemypie** [comment]
@Duncaen how?

File: srcpkgs/kitty/template (hunk 5 above)
Comment (ibrokemypie) [inline srcpkgs/kitty/template]: Can only see a python3-module build style, this isnt actually a module but a C program with python parts.

File: srcpkgs/kitty/template (hunk 6 above)
Comment (ibrokemypie) [inline srcpkgs/kitty/template]: doesnt build using the buildstyles

**Comment by lemmi** [comment]
@DirectorX the glew update at least needs to be a separate commit. Probably even a separate pull request with revbumps for all packages depending on it.

**Comment by DirectorX** [comment]
You are right, this PR should be split


31. Aug 2017 06:17 by notifications@github.com:

**Comment by asergi** [comment]
@ibrokemypie https://github.com/ibrokemypie/dotfiles/blob/master/README.md Still interested in Void? If not, please close all your open PRs here.

**Comment by asergi** [comment]
No response. Closed.

**Pull request closed without merging**